src/guide/index.html

```diff
@@ -104,6 +101,33 @@
 				cm.refresh();
 			}, 3000 );
 		});
+
+		var sidebarOpen = false;
+		var animationEnded = true;
+		var menuLinkEl = document.getElementsByClassName("menu-link")[0];
+		var sidebarEl = document.getElementsByClassName("sidebar")[0];
```

Comment:
Could be achieved quicker with `document.querySelector(".sidebar")`, which by default picks the first result.